pandas/tests/groupby/aggregate/test_aggregate.py

```diff
@@ -1577,3 +1577,9 @@ def test_agg_groupings_selection():
     )
     expected = DataFrame({"b": [6, 4], "c": [11, 7]}, index=index)
     tm.assert_frame_equal(result, expected)
+
+def test_agg_std():
+    df = pd.DataFrame({'A': [1, 2, 3], 'B': [4, 5, 6]})
+    std = df.agg(np.std)
```

Comment:
```suggestion
    expected = df.agg(np.std)
```

Could you also compare `df.agg([np.std])`?

Reply:
Hello @mroeschke, I'm sorry but I'm not sure what you are asking. Do you want me to check the type of class? e.g (pandas.core.series.Series)

Reply:
Just asking to add a test to also check the result of `df.agg([np.std])`  as well